### PR TITLE
[Content] Create script to sync content with a google sheet

### PR DESCRIPTION
**Pull request by Catacola**
## Description of the change

This introduces a new script: `tools/syncContentsFromSheets.ts`. It can be invoked via `yarn sync-contents`.

This script will reach into a google sheet and generate the content blobs. The content in this diff was generated from https://docs.google.com/spreadsheets/d/1NaU5neS0fAsr_ljAOMi05VxXRUBxuAiEQ0ndky8vnkU.

Credentials and sheet ids are stored as env variables in `.env-cmdrc` which will be synced externally via 1password.

## Type of change

- [x] New feature (non-breaking change that adds functionality)

## Related issues

Part of #1264 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally
- [ ] E2E have been run for Lantern changes ([Steps in the Readme](https://github.com/Recidiviz/pulse-dashboard#running-e2e-tests))

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/core/content/page/default.ts [hunk 1]
```diff
@@ -23,26 +23,23 @@ import { PageCopy } from "../types";
 const content: PageCopy = {
   prison: {
     title: "Prison",
-    summary: `This chart includes individuals who are admitted to state facilities,
-      including termers, riders, parole violators, and people held in county jails
-      under state jurisdiction.`,
+    summary:
+      "This chart includes individuals who are admitted to state facilities, including termers, riders, parole violators, and people held in county jails under state jurisdiction.",
```
Comment (Catacola) [inline src/core/content/page/default.ts:27]: The process will currently prettify the files, but long strings will stay on one line. I can look into splitting them in the future.

File: src/core/content/page/us_id.ts [hunk 2]
```diff
@@ -21,19 +21,13 @@
...
-  prison: {
-    summary: `This chart includes individuals who are admitted to IDOC facilities
-      and CAPP, including termers, riders, parole violators, and people held in county
-      jails under IDOC jurisdiction.`,
-  },
   supervision: {
-    summary: `This charts includes all individuals that are currently on probation
-      and/or parole, including those in the Limited Supervision Unit.`,
+    summary:
+      "This charts includes all individuals that are currently on probation and/or parole, including those in the Limited Supervision Unit.",
```
[the remark this answers is not in the document]
Comment (jovergaag) [reply src/core/content/page/us_id.ts:26]: I played around with this quite a bit, looks like google sheets defaults to the straight quotes so no need to worry about that. Double and single quotes are both coming through fine!

**Review by jovergaag — APPROVED** [review]
Super cool. I'm so excited for this!

File: src/core/content/page/us_id.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Catacola) [reply src/core/content/page/us_id.ts:26]: It's so nice when a set of PR comments resolve themselves.

**Pull request merged**